aws/docker-pull.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Intended for use on the initial build of a box. Ansible will update/restart
+# if changed subseqsently.
+
+IMAGES=(
+  jrgm/123done
+  mozilla/fxa-auth-db-mysql
```

Comment:
Setting a tag is a TODO.

I was about to say that it should be fine to pull these here, but the largest layers in our images are the ones that will change most often. So this script also should be taught to handle tags. Good point.